### ISSUE TITLE
Missing message after killing 1st boss in heroics

### ISSUE THREAD

**Issue by Cronicman** [AscEmu/AscEmu#252]
It's missing the message "You are now saved to this instance" after defeating the 1st boss in any heroic.

**Comment by Zyres** [comment]
Only for heroic difficulty or for every instances? What is the function behind this message?

**Comment by Lectro** [comment]
This must work only on heroic dungeons and raids

**Comment by Nupper** [comment]
all versions for raids...Heroic Only for dungeons

**Comment by Zyres** [comment]
Again: What is the function behind the message? It is useless to print this message without any function? What happen if "You are now saved to this instance"?

**Comment by Nupper** [comment]
you would not be able to repeat the Dungeon Heroic for 24 hours.. and you cannot repeat the Raid for a week...your essentially saved and locked out.

**Comment by Zyres** [comment]
@aaron02 I can't remember that issue, is this still a valid issue in AE?

**Comment by aaron02** [comment]
i dont think this is a issue next time when i go into a heroic i look if there is a message ingame

**Comment by Appled** [comment]
It's possible that the message was implemented in wotlk but I'm pretty sure there should be one

**Comment by aaron02** [comment]
the message was implemented in TBC but i only saw this message in Zul Aman when opening the gates

**Comment by Appled** [comment]
> the message was implemented in TBC but i only saw this message in Zul Aman when opening the gates

Have you seen it in heroic dungeons after killing a boss?

**Comment by Appled** [comment]
Fixed in https://github.com/AscEmu/AscEmu/commit/d1f180970f80f00c961f5a96349e33af98728bc6
